src/main/java/core/basesyntax/WriteData.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteData {
+    public void writer(String toFile,String dataToWrite) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFile))) {
+            bufferedWriter.write(dataToWrite);
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t write data to file: ",e);
```

Comment:
seems you've missed the filename

```suggestion
            throw new RuntimeException("Can`t write data to file: " + toFile, e);
```